app/models/concerns/push_updates.rb

```diff
@@ -33,8 +33,8 @@ def update_client_store(operation)
 
   def browse_updates(operation)
     operation = is_browse? ? operation : "delete"
-    json = Api::V1::BrowsePackageSerializer.new(self).as_json[:browse_package]
-    data = { item: { package: json }, operation: operation }
+    json = Api::V1::BrowsePackageSerializer.new(self).as_json
+    data = { item: { package: json[:browse_package], images: json[:images]  }, operation: operation }
```

Comment:
Should the `images` detail go inside the serializer? or perhaps a BrowseItemSerializer that includes `package` and `images` nodes? This would make it easier to extend in the future. What do you think?